packages/react-tweet/src/twitter-theme/tweet-quoted-tweet-media.tsx

```diff
@@ -0,0 +1,67 @@
+import { Fragment } from 'react'
+import clsx from 'clsx'
+import { type EnrichedQuotedTweet, getMediaUrl } from '../utils.js'
+import { MediaDetails } from '../api/index.js'
+import { TweetMediaVideo } from './tweet-media-video.js'
+import { MediaImg } from './media-img.js'
+import s from './tweet-quoted-tweet-media.module.css'
+
+const getSkeletonStyle = (media: MediaDetails, itemCount: number) => {
+  let paddingBottom = 100 // default of 1x1
+
+  // if we have 2 items, use 8x9
+  if (itemCount === 2) paddingBottom = 112.5
+
+  return {
+    width: media.type === 'photo' ? undefined : 'unset',
+    paddingBottom: `${paddingBottom}%`,
+  }
+}
```

Comment:
I might be missing something, but I wouldn't think that this `getSkeletonStyle` method would need to differ at all from the one used in `tweet-media.tsx`. The layout of media should have the same layout as a normal tweet.

That makes me then wonder why we need a separate `tweet-quoted-tweet-media.tsx` at all? The minor styling differences might be able to be accounted for in the main media component.

Reply:
You might be right. I just noticed some mismatch between `react-tweet` image handling and twitter embeds. Take [this](https://publish.twitter.com/?query=https%3A%2F%2Ftwitter.com%2FDeniTechh%2Fstatus%2F1687016629673050112&widget=Tweet) for instance, ratio is 1:1.

Reply:
Also it's behaviour with 4 images is weird, I can leave it as it is for the moment and open another issue for aligning its behaviour with twitter one.